### PR TITLE
TW-941: Fix images on the details page

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

**Review by Boltman92 — APPROVED** [review]

**Review by herkoss — APPROVED** [review]
GJ!

File: src/components/collectible-icon/utils/get-broken-image-size.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const getBrokenImageSize = (isBigSize: boolean) => ({
+  width: isBigSize ? 72 : 38,
```
Comment (Boltman92) [inline src/components/collectible-icon/utils/get-broken-image-size.ts:2]: maybe we need to add `formatSize()` ? is it works properly on different devices?

**Review by Boltman92 — APPROVED** [review]

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_907

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_907

**Pull request merged**